CMakeLists.txt

```diff
@@ -415,11 +415,12 @@ endif()
 # Enable Threads support
 # =============================================================================
 if(NRN_ENABLE_THREADS)
+  # Note that at least GCC requires -pthread to be passed even when using std::thread.
   set(THREADS_PREFER_PTHREAD_FLAG ON)
   find_package(Threads REQUIRED)
-  set(USE_PTHREAD 1)
-else()
-  set(USE_PTHREAD 0)
+  # TODO: better to use a namespaced macro name like NRN_ENABLE_THREADS, leaving this for now to
+  # manage the size of the diff
```

Comment:
I can include this change in the PR too, but maybe better to do the substantive review first before it gets lost in the noise.

Reply:
Let's do it here? `carbon` might have been better to remove in another PR ?

Reply:
The reason `carbon` came in here because there was some `carbon`-specific `pthread` stuff. But I can separate it out if you like?

Reply:
thanks

Reply:
I changed the macro name to be `NRN_ENABLE_THREADS`